app/models/ingredient.rb

```diff
@@ -0,0 +1,11 @@
+class Ingredient < ApplicationRecord
+  has_many :dish_ingredients
+  has_many :dishes, through: :dish_ingredients
+
+  def self.calorie_count
+    sum(:calories)
+  end
```

Comment:
This method doesn't quite do what we were looking for. This method will give the calorie_count of ALL ingredients, but we wanted a method that would give the calories for the ingredients of a particular dish. That to me indicates that this method should live in the Dish model and then be `def total_calories` and the method would be `ingredents.sum(:calories)`